otherlibs/unix/channels_win32.c

```diff
@@ -60,15 +60,36 @@ static DWORD check_stream_semantics(value handle)
   }
 }
 
+int caml_win32_get_CRT_fd(value handle)
```

Comment:
It feels strange to me that we end up with two "get an fd from a file_descr" functions, given the actual usage of `CRT_fd_val` as an rvalue. The suggestions below involve swapping the two values of `GETTING_CRT_FD` and `NO_CRT_FD` so that this function isn't needed.

Reply:
I'd rather keep the function and remove the macro.